From 3488f8579f578996f93dfa39b9eaa9fd4b1cbcdf Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Dec 2024 17:54:37 +0800
Subject: [PATCH 169/261] AOSCOS: input: atkbd: disable
 KEYBOARD_ATKBD_LEMOTE_KEYCODES by default

This option sets a special keycode to workaround issues with certain
multimedia Fn key combos on the Lemote A1201 laptop but does not use DMI
to match device models.

Disable this option by default until we adopt DMI detect-and-match to this
particular option.

Fixes: "AOSCOS: MIPS: Loongson: Add the multifunction keys (Fnkey) support"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/input/keyboard/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/keyboard/Kconfig b/drivers/input/keyboard/Kconfig
index dd127dac0b78..ac2612f2fa7d 100644
--- a/drivers/input/keyboard/Kconfig
+++ b/drivers/input/keyboard/Kconfig
@@ -159,7 +159,7 @@ config KEYBOARD_ATKBD_RDI_KEYCODES
 config KEYBOARD_ATKBD_LEMOTE_KEYCODES
 	bool "Use Lemote laptop (A1201) keyboard scancodes"
 	depends on MACH_LOONGSON64 && KEYBOARD_ATKBD
-	default y
+	default n
 	help
 	  Say Y here if you want to use an AT or PS/2 keyboard, and your
 	  keyboard uses keycodes that are specific to Lemote laptop (A1201)
-- 
2.48.1

